### PR TITLE
Happy Thoughts Machine

### PR DESCRIPTION
**Pull request by pwangy**
Link to my deployed project: 
https://pwangy-happy.netlify.app/

My reflections on how this week's project turned out: 
One of my goals this week was to finish this project early so I could have dedicated time to rework my portfolio. So I'm happy to have completed the blue level goals AND hit both a red and a black level goal too. This week also felt really good in that a few React concepts are _finally_ starting to make more sense to me through repetitive use in a few different projects. Starting this little app was fun and I think I got really close in cloning the styling of the app too. Then I hit the "Like" button functionality and that was the hardest bit to figure out this week. After struggling with that for a while, I was finally able to make it work. I don't completely understand it, but I think it will make more sense, the more I practice. 

Things I'd like to have clarified or explained in more detail: 
Should we be breaking out our CSS into different files to reflect our components? Does it matter?

### CODE REVIEW

**Review by MindstormingAB — CHANGES_REQUESTED** [review]
Well done Peggy! It was really interesting to dive into your code. You showed a good understanding of the React states and achieved easily the blue requirements. I'm really impressed by your local storage and the local count of likes.
Next time, you could improve readability by creating more components and you could work more on accessibility.
Before I approve your pull request, I'd like you to make sure that the page works on mobile phones.

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+import { Footer } from './components/Footer'
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="app-container">
+    <h1>Happy Thoughts Machine <span 
+          role="img"
+          aria-label="love letter emoji"
```
[the remark this answers is not in the document]
Comment (pwangy) [reply code/src/App.js:12]: added!

File: code/.eslintrc.json [hunk 2]
```diff
@@ -122,4 +122,4 @@
     "react-hooks/exhaustive-deps": "warn",
     "semi": "off"
```
[the remark this answers is not in the document]
Comment (pwangy) [reply code/.eslintrc.json:123]: I wish I could! I think this updated when I went through Van's steps to update whatever npm was complaining about.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
+## Requirements Completed
+🔵  Blue Level
+- The page should follow the design as closely as possible
+- It should list the most recent thoughts at the top and older thoughts at the bottom (sorted)
+- The app should show the content of the message and how many likes they've received
+- The app should have a form to post new thoughts
+- Implement the heart button to send likes on a thought
+
+🔴  Red Level (Intermediary Goals)
+- Show a count below the form input that updates as the user types and shows how many characters are remaining. Make it go red when the user has typed over 140 characters
```
[the remark this answers is not in the document]
Comment (pwangy) [reply README.md:34]: :D

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,191 @@
...
+  border: 2px solid #4d4d4d;
+  display: flex;
+  flex-flow: column nowrap;
 }
+
+.post-txt {
+  margin: 16px 3px;
+  padding-bottom: 26px;
+  font-size: 16px;
+  word-break: break-all;
```
[the remark this answers is not in the document]
Comment (pwangy) [reply code/src/index.css:113]: > You should use `word-break: break-word` so you can avoid this
> image

Sandrine, what browser are you using? I haven't been able to reproduce this error on my machine. :(

**Pull request closed without merging**